rir/src/compiler/test/PirCheck.cpp

```diff
@@ -135,10 +135,11 @@ static bool testOneNot(ClosureVersion* f) {
     return numNots == 1;
 }
 
-static bool testLdVarInFirstBB(ClosureVersion* f) {
+static bool testLdVarVectorInFirstBB(ClosureVersion* f) {
     for (auto instruction : *f->entry) {
-        if (LdVar::Cast(instruction)) {
-            return true;
+        if (auto ldvar = LdVar::Cast(instruction)) {
+            return std::string(CHAR(PRINTNAME(ldvar->varName)))
```

Comment:
variable names are symbols. should be compared by:
`ldvar->varName == Rf_install("vector")`